rpc/accounts.go

```diff
@@ -45,7 +45,12 @@ func (s *RPC) listAccounts(
 			out[i].Email = &dbAcc.Email
 		}
 	}
-	return out, nil
+
+	res := &intents.IntentResponseAccountList{
+		Accounts:         out,
+		CurrentAccountID: sess.Identity,
```

Comment:
i wonder if we should rename this to, `SessionAccountID` ..? but `CurrentAccountID` is fine too.. whatever you guys prefer

Reply:
I'd be wary of using the word `session`, we have way too many "sessions" and it's now a very overloaded word IMO 😅 though maybe in this case it'd fit well? not 100% sure

Reply:
cool, lets stick to CurrentAccountID :)